cluster-provision/k8s/1.31/extra-pre-pull-images

```diff
@@ -8,3 +7,5 @@ quay.io/calico/cni:v3.18.0
 quay.io/calico/kube-controllers:v3.18.0
 quay.io/calico/node:v3.18.0
 quay.io/calico/pod2daemon-flexvol:v3.18.0
+quay.io/kubevirtci/grafana-grafana:11.1.0
+quay.io/prometheus-operator/prometheus-config-reloader:v0.75.1
```

Comment:
It looks like there is an older version of this image - https://github.com/kubevirt/kubevirtci/pull/1228/files#diff-f4afa2d47e15ef77f00b37aa0b6308240dd8923fd1a933c12dacf09f7bf48b17R5

Reply:
@brianmcarey should i create the different commit for each provider  or append all the changes to the present coommit

Reply:
Different commits for each provider would be good.

Reply:
@brianmcarey thank you